core/rawdb/table.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
Similar points as in `database.go` about whether this is the right package for this to live in. However, this is more complicated as `Table` contains the prefix for where particular data is stored. This makes have a clear separation of abstractions more difficult.